engine/runtime/src/main/java/org/enso/interpreter/node/expression/builtin/number/bigInteger/AddNode.java

```diff
@@ -0,0 +1,30 @@
+package org.enso.interpreter.node.expression.builtin.number.bigInteger;
+
+import com.oracle.truffle.api.dsl.Specialization;
+import com.oracle.truffle.api.nodes.Node;
+import org.enso.interpreter.dsl.BuiltinMethod;
+import org.enso.interpreter.node.expression.builtin.number.utils.BigIntegerOps;
+import org.enso.interpreter.node.expression.builtin.number.utils.ToEnsoNumberNode;
+import org.enso.interpreter.runtime.number.EnsoBigInteger;
+
+@BuiltinMethod(type = "Big_Integer", name = "+", description = "Big integer addition.")
+public abstract class AddNode extends Node {
+  private @Child
+  ToEnsoNumberNode toLongNode = ToEnsoNumberNode.build();
```

Comment:
This is a very minor thing, but this naming became inconsistent - the `ToEnsoNumberNode` is now sometimes stored as `toLongNode` and sometimes as `toEnsoNumberNode` while this is the same thing. I think this may possibly confuse future readers.